### PR TITLE
Update links in README

### PR DESCRIPTION
**Pull request by dolph**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -132,8 +132,8 @@ Congratulations! You successfully created your Bluemix account and installed the
 2. Before you continue to the next step, verify that the deployment of your worker node is complete.
     ```bash
     $ bx cs workers [your_cluster_name]
-    ID                                           Public IP       Private IP    Machine Type  State     Status   
-    dal10-pa8dfcc5223804439c87489886dbbc9c07-w1  169.47.223.113  10.171.42.93  free         deployed  Deploy Automation                                          Successful   
+    ID                                           Public IP       Private IP    Machine Type  State     Status
+    dal10-pa8dfcc5223804439c87489886dbbc9c07-w1  169.47.223.113  10.171.42.93  free         deployed  Deploy Automation                                          Successful
```
[the remark this answers is not in the document]
Comment (dolph) [reply README.md]: It is. It [renders correctly](https://github.com/IBM/container-journey-template/pull/17/commits/7665f9721442e2cdb4a2d771dece5bdc7d00f7a4?short_path=04c6e90#diff-04c6e90faac2675aa89e2176d2eec7d8) as tab-delimited columns, but all I did was remove trailing whitespace.

[the remark this answers is not in the document]
Comment (MHBauer) [reply README.md]: Speaking of rendering, how about this?

screenshot from 2019-01-23 11-23-51

**Comment by MHBauer** [comment]
It looks like I can merge this, any objections?

File: README.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (dolph) [reply README.md]: @MHBauer here's a fix for that issue [here](https://github.com/IBM/container-journey-template/pull/18).

**Pull request merged**